code/modules/food_and_drink/cakes.dm

```diff
@@ -1,3 +1,8 @@
+//defines used later for custom cake utility procs
+#define CAKE_MODE_CAKE 1
+#define CAKE_MODE_SLICE 2
+#define CAKE_MODE_STACK 4
+#define CAKE_MODE_BUILD 9
```

Comment:
are these bitflags or different numerical modes?
the numbers make sense for neither

Reply:
they're literally just for readability, originally they were just numbers with no meaning other than letting the code know which part of very specific functions to use

Reply:
so, different numerical modes? why not just 1,2,3,4 then

Reply:
that works!